app/models/in_progress_form.rb

```diff
@@ -110,6 +109,14 @@ def expires_after
                        end
   end
 
+  def reset_expires_at!
+    skippable_forms = %w[21P-527EZ 21-526EZ 5655]
+    return if skippable_forms.include?(form_id)
+
+    self.expires_at = Time.current + expires_after
+    save unless new_record?
```

Comment:
Why dont we save when it is a new record? Is it because we know for sure there is/will be a save coming further down the line in all cases when it is a new record?